MatrixSDK/Utils/MXHTTPClient.m

```diff
@@ -81,6 +86,17 @@ -(id)initWithBaseURL:(NSString *)baseURL andAccessToken:(NSString *)access_token
         
         httpManager = [[AFHTTPRequestOperationManager alloc] initWithBaseURL:[NSURL URLWithString:baseURL]];
         
+        // Check whether some certificates have been included in app bundle
+        AFSecurityPolicy *securityPolicy = [AFSecurityPolicy policyWithPinningMode:AFSSLPinningModeCertificate];
+        if (securityPolicy.pinnedCertificates.count)
```

Comment:
Looks like this is AFNetworking looking for certs in the app bundle - do we need this? Or does it look in the Documents directory too?